lib/libxdp/tests/test-libxdp.sh

```diff
@@ -102,6 +102,16 @@ test_xsk_non_privileged()
 	ip link delete xdp_veth0
 }
 
+test_link_dispatch()
+{
+        if test ! -f $TEST_PROG_DIR/test_link_dispatch; then
+		exit "$SKIPPED_TEST"
+	fi
+	ip link add xdp_veth0 type veth peer name xdp_veth1
+	check_run $TESTS_DIR/test_link_dispatch xdp_veth0
+	ip link delete xdp_veth0
```

Comment:
This should be added to `cleanup_tests()` below as well

Reply:
test cases b4 this one deleting at the end as well ?